charts/sessionspaces/templates/clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "sessionspaces.fullname" . }}
+rules:
+- apiGroups: [""]
+  resources: ["pods", "services"]
+  verbs: ["get", "list", "watch", "patch", "create"] 
```

Comment:
Does sessionspaces need permission to operate on `pods` or `services`?

Reply:
It creates a argo-workflows clusterrole and has rules for pods, but not for services, I will update it.